analytics-webapps/src/main/webapp/js/statistic-collection.js

```diff
@@ -95,7 +96,8 @@ function() {
         'operation': bookmark && 'Bookmark' || 'UnBookmark',
         'timestamp': Date.now(),
         'parameters': {
-          'type': eventDetail.typeLabel || eventDetail.type,
+          'type': type,
+          'dataType': `${type.toUpperCase()}_TYPE`,
```

Comment:
```suggestion
          type,
          'dataType': type.toLowerCase(),
```

It would be better to conserve the lowercase of data type, same as object type but case insensitive.

Reply:
Done!